html-css/README.md

```diff
@@ -67,3 +67,14 @@ A mighty, modern linter that helps you avoid errors and enforce conventions in y
 4. Run `npx stylelint "**/*.{css,scss}"`.
 5. Fix linter errors.
 6. **IMPORTANT NOTE**: feel free to research [auto-correct options for Stylelint](https://stylelint.io/user-guide/cli#autofixing-errors) if you get a flood of errors but keep in mind that correcting style errors manually will help you to make a habit of writing a clean code!
+
+## Have you already completed the Sass lessons?
+
+If you completed the Sass lessons you may prefer to work with SCSS files, if that's the case you need to perform
+a few modifications to the linter configuration files.
+
+1. In the file `.github/workflows/linters.yml` line 48 replace `"**/*.{css,scss}"` with `"**/*.scss"`
```

Comment:
We can add a link to that line to make it more explicit:

```
1. In the file `.github/workflows/linters.yml` [line 48](https://github.com/microverseinc/linters-config/blob/master/html-css/.github/workflows/linters.yml#L48) replace `"**/*.{css,scss}"` with `"**/*.scss"`
```

Reply:
I like that better.